pkg/store/github/github.go

```diff
@@ -250,30 +252,48 @@ func (s *GitHubStore) ReloadProviderCache(ctx context.Context) error {
 
 		var versions []core.ProviderVersion
 		for _, release := range releases {
-			var platforms []core.Platform
 			version := strings.TrimPrefix(release.GetName(), "v")
 
+			if s.providerIgnoreCache[cacheKey(nameKey, version)] {
+				s.logger.Debug(fmt.Sprintf("Ignoring release [%s/%s], previously found to be not valid", nameKey, version))
+				continue
+			}
+
+			var platforms []core.Platform
```

Comment:
what is this?

Reply:
Not relevant for the change in this PR. I just moved it a bit down. Moved it back up again.